index.html

```diff
@@ -0,0 +1 @@
+<h1 class="welcome">Hello Microverse</h1>    
```

Comment:
- [OPTIONAL] I marked this optional because it was not mentioned by the previous reviewers. You did not get the generic skeleton for your index.html properly. Below is how the content of your `index.html` should look like.
```
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial- 
        scale=1.0">
        <title>Document</title>
        <link rel="stylesheet" href="style.css"> 
    </head>
    <body>
        <h1 class="welcome">Hello Microverse</h1>
    </body>
</html>
``` 
You need to link your `style.css` file to your `index.html` before whatever style you give to the content of the `<h1>` could take effect. Kindly follow this step and see the color of `Hello Microverse` would change to orange color you gave in the css file.